### PR TITLE
Runtime 6.0.3 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Once the changes are approved in Astro, I'll push the notes to Software and Nebula

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -11,6 +11,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+Astro Runtime 6.0.2 includes same-day support for Airflow 2.4.2, which includes a collection of bug fixes. Fixes include:
+
+- Handle mapped tasks in task duration chart ([#26722](https://github.com/apache/airflow/pull/26722))
+- Make tracebacks opt-in ([#27059](https://github.com/apache/airflow/pull/27059)
+
+For a complete list of commits, see the [Apache Airflow 2.4.2 release notes](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-4-2-2022-10-23.
+
+### Additional improvements  
+
+- Upgraded `openlineage-airflow` to 0.15.1, which aa dedicated development environment for Airflow. You can now continually make and test changes to custom OpenLineage extractors in an Airflow environment without needing to rebuild your Docker images. For more information, see the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md).
```
Comment (collinmcnulty) [inline astro/runtime-release-notes.md:30]: aa?

Comment (kwcanuck) [inline astro/runtime-release-notes.md:30]:
```suggestion
- Upgraded `openlineage-airflow` to 0.15.1, which is a dedicated Airflow development environment. You can now create and test changes to custom OpenLineage extractors in an Airflow environment without needing to rebuild your Docker images. For more information, see the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md).
```

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -11,6 +11,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Astro Runtime 6.0.3
+
+- Release date: October 24, 2022
+- Airflow version: 2.4.2
+
+### Airflow 2.4.1
+
+Astro Runtime 6.0.2 includes same-day support for Airflow 2.4.2, which includes a collection of bug fixes. Fixes include:
```
Comment (kwcanuck) [inline astro/runtime-release-notes.md:21]:
```suggestion
Astro Runtime 6.0.2 includes same-day support for Airflow 2.4.2 and the following bug fixes:
```

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -11,6 +11,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
 
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Astro Runtime 6.0.3
+
+- Release date: October 24, 2022
+- Airflow version: 2.4.2
+
+### Airflow 2.4.1
```
Comment (kwcanuck) [inline astro/runtime-release-notes.md:19]:
```suggestion
### Airflow 2.4.2
```

**Review by kwcanuck — APPROVED** [review]

**Review by merobi-hub — CHANGES_REQUESTED** [review]
Thanks, @jwitz, the description of OpenLineage 0.15.1 looks good. The only change I would make would be to say "which includes a dedicated..." on line 30.

File: astro/runtime-release-notes.md [hunk 4]
```diff
@@ -11,6 +11,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+Astro Runtime 6.0.2 includes same-day support for Airflow 2.4.2 and the following bug fixes:
+
+- Handle mapped tasks in task duration chart ([#26722](https://github.com/apache/airflow/pull/26722))
+- Make tracebacks opt-in ([#27059](https://github.com/apache/airflow/pull/27059)
+
+For a complete list of commits, see the [Apache Airflow 2.4.2 release notes](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-4-2-2022-10-23.
+
+### Additional improvements  
+
+- Upgraded `openlineage-airflow` to 0.15.1, which is a dedicated Airflow development environment. You can now create and test changes to custom OpenLineage extractors in an Airflow environment without needing to rebuild your Docker images. For more information, see the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md).
```
Comment (jwitz) [inline astro/runtime-release-notes.md:30]:
```suggestion
- Upgraded `openlineage-airflow` to 0.15.1, which includes a dedicated Airflow development environment. You can now create and test changes to custom OpenLineage extractors in an Airflow environment without needing to rebuild your Docker images. For more information, see the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md).
```

File: astro/runtime-release-notes.md [hunk 5]
```diff
@@ -11,6 +11,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Astro Runtime 6.0.3
+
+- Release date: October 24, 2022
+- Airflow version: 2.4.2
+
+### Airflow 2.4.2
+
+Astro Runtime 6.0.3 includes same-day support for Airflow 2.4.2.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:21]:
```suggestion
Astro Runtime 6.0.3 includes same-day support for Airflow 2.4.2. Some changes in Airflow 2.4.2 include:
```

File: astro/runtime-release-notes.md [hunk 6]
```diff
@@ -11,6 +11,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+- Release date: October 24, 2022
+- Airflow version: 2.4.2
+
+### Airflow 2.4.2
+
+Astro Runtime 6.0.2 includes same-day support for Airflow 2.4.2 and the following bug fixes:
+
+- Handle mapped tasks in task duration chart ([#26722](https://github.com/apache/airflow/pull/26722))
+- Make tracebacks opt-in ([#27059](https://github.com/apache/airflow/pull/27059)
+
```
[the remark this answers is not in the document]
Comment (jwitz) [reply astro/runtime-release-notes.md:25]: I think the last edit I made removed some of the context from this list. We are just highlighting a few changes in 2.4.2 that might be of interest to people reading the notes. Tried to make that more clear in the preceding sentence.

**Pull request merged**